launcher/package/rpm/MultiMC5.spec

```diff
@@ -23,22 +29,29 @@ mkdir -p %{buildroot}/opt/multimc
 install -m 0644 ../ubuntu/multimc/opt/multimc/icon.svg %{buildroot}/opt/multimc/icon.svg
 install -m 0755 ../ubuntu/multimc/opt/multimc/run.sh %{buildroot}/opt/multimc/run.sh
 mkdir -p %{buildroot}/%{_datadir}/applications
-install -m 0644 ../ubuntu/multimc/usr/share/applications/multimc.desktop %{buildroot}/%{_datadir}/applications/multimc.desktop
+desktop-file-install --dir=%{buildroot}%{_datadir}/applications ../ubuntu/multimc/usr/share/applications/multimc.desktop
+
 mkdir -p %{buildroot}/%{_datadir}/metainfo
-install -m 0644 ../ubuntu/multimc/usr/share/metainfo/multimc.metainfo.xml %{buildroot}/%{_datadir}/metainfo/multimc.metainfo.xml
+install -m 0644 ../ubuntu/multimc/usr/share/metainfo/multimc.metainfo.xml %{buildroot}/%{_metainfodir}/multimc.metainfo.xml
 mkdir -p %{buildroot}/%{_mandir}/man1
 install -m 0644 ../ubuntu/multimc/usr/share/man/man1/multimc.1 %{buildroot}/%{_mandir}/man1/multimc.1
 
+%check
+appstream-util validate-relax --nonet %{buildroot}%{_metainfodir}/multimc.metainfo.xml
+
 %files
 %dir /opt/multimc
 /opt/multimc/icon.svg
 /opt/multimc/run.sh
 %{_datadir}/applications/multimc.desktop
-%{_datadir}/metainfo/multimc.metainfo.xml
-%dir /usr/share/man/man1
-%{_mandir}/man1/multimc.1.gz
+%{_metainfodir}/multimc.metainfo.xml
+%dir %{_mandir}/man1
+%{_mandir}/man1/multimc.1*
```

Comment:
It is recommended to use a wildcard at the end since rpm is in charge of compressing the man page and in the future this might change from gzip to something else    
https://docs.fedoraproject.org/en-US/packaging-guidelines/#_manpages